packages/frames.js/src/middleware/images-worker/next/index.tsx

```diff
@@ -60,13 +60,17 @@ export function createImagesWorker(
       const width =
         aspectRatio === "1.91:1" ? Math.round(height * 1.91) : height;
 
-      return new ImageResponse(
-        <Scaffold>{jsx}</Scaffold>,
-        options.imageOptions || {
-          width,
-          height,
-        }
-      ) as Response;
+      const {
+        width: overrideWidth,
+        height: overrideHeight,
+        ...overrideOptions
+      } = options.imageOptions || {};
+
+      return new ImageResponse(<Scaffold>{jsx}</Scaffold>, {
+        width: overrideWidth || width,
+        height: overrideHeight || height,
+        ...overrideOptions,
```

Comment:
this double overriding is hard to reason about and I think unnecessarily so - there's a simpler way to express this

```ts
const imageOptions = { width, height, ...options.imageOptions, ...overrideOptions };

return new ImageResponse(<Scaffold>{jsx}</Scaffold>, imageOptions);
```

Reply:
requires an appropriate default value of `{}` in the parameters for overrideOptions & options.imageOptions

Reply:
it's tricky because the user shouldn't actually be able to set the width and height to fixed values since those are dependent on the aspect ratio. might make more sense for the options to allow a user to set a width and height for a given aspect ratio

Reply:
yeah not sure of the details, up to you